src/modules/logger/logged_topics.cpp

```diff
@@ -69,6 +69,7 @@ void LoggedTopics::add_default_topics()
 	add_topic("offboard_control_mode", 1000);
 	add_topic("position_controller_status", 500);
 	add_topic("position_setpoint_triplet", 200);
+	add_topic("rpm", 500);
```

Comment:
Can you keep the listed sorted?

Reply:
I'm sorry @dagar. I'll be more careful, next time. Now it is sorted.

Reply:
Not a big deal, thanks for the quick response.